configuration/scripts/cice.settings

```diff
@@ -12,7 +12,7 @@ setenv ICE_OBJDIR     ${ICE_RUNDIR}/compile
 setenv ICE_RSTDIR     ${ICE_RUNDIR}/restart
 setenv ICE_HSTDIR     ${ICE_RUNDIR}/history
 setenv ICE_LOGDIR     ${ICE_CASEDIR}/logs
-setenv ICE_DRVOPT     cice
+setenv ICE_DRVOPT     standalone/cice
```

Comment:
@apcraig I noticed that this change from `drivers/cice` to `drivers/standalone/cice` was not reflected in the cice.settings documentation, that still references `cice`. Minor, but still thought I'd mention it.

Reply:
Good catch, I'll update it in a sandbox that I plan to PR soon.  Thanks!